.travis.yml

```diff
@@ -1,6 +1,8 @@
 sudo:     required
 language: go
 go:       1.7.5
-
+services: redis-server
+env:
+  - REDIS_SOCKET_TYPE=tcp REDIS_URL="localhost:6379"
 install: make bootstrap
-script:  make check_format tests
+script:  make check_format tests_integration
```

Comment:
ok:
 `tests_unit` runs only the unit tests and
 `tests` runs all the tests unit + integration.